mibs/COMAP-InteliLite4-29.txt

```diff
@@ -0,0 +1,8718 @@
+COMAP-InteliLite4-29 DEFINITIONS ::= BEGIN

```

Comment:
Please remove .txt from the filename

Reply:
.txt is now removed. Also changed to lower case filename